templates/drupal/1/rancher-compose.yml

```diff
@@ -0,0 +1,33 @@
+.catalog:
+  name: "Drupal"
+  version: "0.1.4"
+  description: "Blog tool, publishing platform and CMS"
+  uuid: Drupal-0
+  minimum_rancher_version: v0.51.0
+  questions:
+    - variable: public_port
+      description: "public port to access the drupal site"
+      label: "Public Port"
+      required: true
+      default: "15080"
+      type: "int"
+    - variable: "DB_NAME"
+      description: "Database Name to use for the server"
+      label: "DB Name"
+      required: true
+      default: "drupal"
+      type: "string"
+    - variable: "DB_USER"
+      description: "Database User to use for the server"
+      label: "DB User"
+      required: true
+      default: "drupal"
+      type: "string"
+    - variable: "DB_PASS"
+      description: "Database Password to use for the server"
+      label: "DB Password"
+      required: true
+      default: ""
+      type: "string"
```

Comment:
Only difference with version 0.1.3 (password)....
Could you please merge versions into one??